$MirOS: ports/mailnews/pine/patches/patch-pine_strings_c,v 1.1 2005/05/04 23:37:41 tg Exp $
--- pine/strings.c.orig	Tue Aug 30 00:07:57 2005
+++ pine/strings.c	Fri Oct  7 19:12:56 2005
@@ -1468,7 +1468,7 @@ comatose(number) 
 	if(x != 0 || done_one) {
 	    if(b != buf[whichbuf])
 	      *b++ = ',';
-	    sprintf(b, done_one ? "%03ld" : "%d", x);
+	    sprintf(b, done_one ? "%03ld" : "%ld", x);
 	    b += strlen(b);
 	    done_one = 1;
 	}
@@ -4997,7 +4997,7 @@ open_any_patterns(rflags)
 {
     long canon_rflags;
 
-    dprint(7, (debugfile, "open_any_patterns(0x%x)\n", rflags));
+    dprint(7, (debugfile, "open_any_patterns(0x%lx)\n", rflags));
 
     canon_rflags = CANONICAL_RFLAGS(rflags);
 
@@ -5129,7 +5129,7 @@ close_patterns(rflags)
 {
     long canon_rflags;
 
-    dprint(7, (debugfile, "close_patterns(0x%x)\n", rflags));
+    dprint(7, (debugfile, "close_patterns(0x%lx)\n", rflags));
 
     canon_rflags = CANONICAL_RFLAGS(rflags);
 
@@ -5199,7 +5199,7 @@ any_patterns(rflags, pstate)
     int  ret = 0;
     long canon_rflags;
 
-    dprint(7, (debugfile, "any_patterns(0x%x)\n", rflags));
+    dprint(7, (debugfile, "any_patterns(0x%lx)\n", rflags));
 
     memset((void *)pstate, 0, sizeof(*pstate));
     pstate->rflags    = rflags;
@@ -7374,7 +7374,7 @@ write_patterns(rflags)
     int canon_rflags;
     int err = 0;
 
-    dprint(7, (debugfile, "write_patterns(0x%x)\n", rflags));
+    dprint(7, (debugfile, "write_patterns(0x%lx)\n", rflags));
 
     canon_rflags = CANONICAL_RFLAGS(rflags);
 
